src/components/Reports/GoalCalculator/SharedComponents/GoalCalculatorGrid/GoalCalculatorGrid.tsx

```diff
@@ -0,0 +1,276 @@
+import React, { useState } from 'react';
+import AddIcon from '@mui/icons-material/Add';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { Box, Button, Card, Switch, Typography, styled } from '@mui/material';
+import {
+  GridActionsCellItem,
+  GridColDef,
+  GridValidRowModel,
+} from '@mui/x-data-grid';
+import { Form, Formik, FormikProps } from 'formik';
+import { useTranslation } from 'react-i18next';
+import * as yup from 'yup';
+import { useGoalCalculator } from '../../Shared/GoalCalculatorContext';
+import { StyledGrid } from './StyledGrid';
+
+const StyledCard = styled(Card)({
+  borderRadius: '4px',
```

Comment:
We can pull the border radius from the theme.

```suggestion
  borderRadius: theme.shape.borderRadius
```